prac_05/wimbledon.py

```diff
@@ -0,0 +1,54 @@
+"""
+Word Occurrences
```

Comment:
Copy and paste?

Reply:
I looked at one of the 3 examples from the lecture slides with the same problem and rewrote that for my first loop. Please see my updated code where I've demonstrated the LBYL method (originally EAFP) for my first loop and a list comprehension for my second loop.